ecologits/tracers/utils.py

```diff
@@ -33,8 +36,7 @@ def llm_impacts(
 
     model = models.find_model(provider=provider, model_name=model_name)
     if model is None:
-        # TODO: Replace with proper logging
-        print(f"Could not find model `{model_name}` for {provider} provider.")
+        logger.warning(f"Could not find model `{model_name}` for {provider} provider.")
```

Comment:
I think the "warning" level is excessive and will still be spammy as exposed in #65. We should probably have an "errors" attribute in the "Impacts" object in the response instead of a logging message.

For now, I would just switch to `logger.debug`.